src/label/creator.py

```diff
@@ -271,6 +281,10 @@ def get_font_size_estimation(text):
     return get_font_size_estimation_from_lookup_table(len(text))
 
 
+def get_label_height_in_px(label_height_mm):
```

Comment:
Does this function only work for the height (the name implies it)? I suspect it might work for width also, and thus can be renamed to `get_mm_in_px()`/`label_mm_to_px()` or similar.

Reply:
You are right, but width is hard-coded, might have a use for positioning of stuff in the future though.